zerver/openapi/zulip.yaml

```diff
@@ -10505,7 +10502,12 @@ paths:
         works, avoids clients needing to worry about large classes of
         potentially messy races, etc.
 
-        **Changes**: The `email_address_visibility` setting was removed in Zulip 7.0 (feature level 163).
+        **Changes**: The `realm_community_topic_editing_limit_seconds` property was
+        removed from the response in Zulip 7.0 (feature level 183) since time limit
+        for editing topics is controlled by `move_messages_within_stream_limit_seconds`
+        setting starting with Zulip 7.0 (feature level 162).
```

Comment:
I generally prefer starting with the feature level, so it doesn't get hidden in the note. Also, we could revise this a bit for the idea of this being the reverse of the changelog note.

> **Changes**: Before Zulip 7.0 (feature level 183), the `realm_community_topic_editing_limit_seconds` property was returned by the response. It was removed because it had not been in use since the realm setting `move_messages_within_stream_limit_seconds` was introduced in feature level 162.

Reply:
I updated this.